cmd/grafana-app-sdk/project.go

```diff
@@ -78,13 +87,18 @@ func setupProjectCmd() {
 	projectAddComponentCmd.Flags().String("grouping", kindGroupingKind, `Kind go package grouping.
 Allowed values are 'group' and 'kind'. This should match the flag used in the 'generate' command`)
 
+	projectLocalGenerateCmd.Flags().Bool("useoldmanifestkinds", false, "Whether to use the legacy manifest style of 'kinds' in the manifest, and 'versions' in each kind. This is a deprecated feature that will be removed in a future release.")
+	projectLocalGenerateCmd.Flags().Lookup("useoldmanifestkinds").NoOptDefVal = "true"
```

Comment:
is this going to be a lot to maintain? i'm wondering if it's easier to update the apps that use it in grafana one PR instead of adding a new flag and delaying the update